### PR TITLE
Toolbar APIs

### PR DESCRIPTION
**Pull request by dcupidon**
Exposed for Android and iOS:
* `bottomToolbar`
* `initialToolbar`
* `hideToolbarsOnTap`

Exposed for iOS:
* `topAppNavBarRightBar`

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -1213,6 +1240,27 @@ Defines whether to hide the bottom toolbar for the current viewer.
...
+#### bottomToolbar
+array of [`Buttons`](./lib/constants.dart) constants, defaults to none.
+
+Defines a custom bottom toolbar. If passed in, the default bottom toolbar will not be used. 
+
+Below is the list of supported buttons for each platform:
+
+| `Button` | Android | iOS |
+| :----- | :-----: | :-----: |
+| `listsButton` | ✅ | ✅ |
```
Comment (sgong-pdftron) [inline API.md:1252]: hmm is this list tested? i don't think share/view mode etc work on android

File: API.md [hunk 2]
```diff
@@ -1213,6 +1240,27 @@ Defines whether to hide the bottom toolbar for the current viewer.
...
+| `thumbnailsButton` | ✅ | ✅ |
+| `shareButton` | ✅ | ✅ |
+| `viewControlsButton` | ✅ | ✅ |
+| `reflowModeButton` | ✅ | ✅ |
+| `searchButton` | ✅ | ✅ |
```
Comment (sgong-pdftron) [inline API.md:1257]: i assume this list is fully tested and everything works?

Comment (dcupidon) [reply API.md:1257]: Yes, the list has been tested

**Review by sgong-pdftron — APPROVED** [review]

File: ios/Classes/PTFlutterDocumentController.m [hunk 3]
```diff
@@ -1050,6 +1050,39 @@ - (void)applyToolGroupSettings
...
+
+        PTToolGroup *matchedDefaultGroup = [self toolGroupForKey:self.initialToolbar toolGroupManager:toolGroupManager];
+        if (matchedDefaultGroup != nil) {
+           // use a default group if its key is found
+           [toolGroupManager setSelectedGroup:matchedDefaultGroup];
+           [self.toolGroupIndicatorView.button setTitle:matchedDefaultGroup.title forState:UIControlStateNormal];
+           if (@available(iOS 13.0, *)) {
+               self.toolGroupIndicatorView.button.largeContentImage = matchedDefaultGroup.image;
+           }
+           return;
```
Comment (JamesTron) [inline ios/Classes/PTFlutterDocumentController.m:1077]: This `return` looks incorrect here.

**Review by JamesTron — CHANGES_REQUESTED** [review]

**Review by JamesTron — APPROVED** [review]

**Pull request merged**